docs/quickstart.md

```
@@ -3,7 +3,64 @@
+Point VibePod at the Podman socket by setting `DOCKER_HOST`:
+
+```bash
+export DOCKER_HOST=unix:///run/user/$(id -u)/podman/podman.sock
+```
+
+!!! warning "Container DNS required"
+    VibePod routes agent traffic through a `vibepod-proxy` container. The agent
+    resolves the proxy by container name over the `vibepod-network` network,
+    which requires **DNS** to be enabled.
+
+    If your Podman installation uses the **CNI** network backend, you must
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)